deploy/scripts/src/accounts/test.ts

```diff
@@ -216,459 +267,365 @@ const phoneData = {
 
   sleepBetween(1, 3)
 
-  res = group('B04_DeleteAccount_04_EnterCurrentPassword POST', () =>
-    timeRequest(() =>
-      res.submitForm({
-        formSelector: "form[action='/enter-password']",
-        fields: {
-          requestType: 'deleteAccount',
-          password: credentials.currPassword
-        },
-        params: {
-          tags: { name: 'B04_DeleteAccount_04_EnterCurrentPassword' }
-        }
-      }),
-    { isStatusCode200, ...pageContentCheck('Are you sure you want to delete your GOV.UK One Login') }))
+  res = group(groups[3], () => timeRequest(() => // B04_DeleteAccount_04_EnterCurrentPassword
+    res.submitForm({
+      formSelector: "form[action='/enter-password']",
+      fields: {
+        requestType: 'deleteAccount',
+        password: credentials.currPassword
+      }
+    }),
+  { isStatusCode200, ...pageContentCheck('Are you sure you want to delete your GOV.UK One Login') }))
 
   sleepBetween(1, 3)
 
-  res = group('B04_DeleteAccount_05_DeleteAccountConfirm POST', () =>
-    timeRequest(() =>
-      res.submitForm({
-        formSelector: "form[action='/delete-account']",
-        params: {
-          tags: { name: 'B04_DeleteAccount_05_DeleteAccountConfirm' }
-        }
-      }),
-    { isStatusCode200, ...pageContentCheck('You have signed out') }))
+  res = group(groups[4], () => timeRequest(() => // B04_DeleteAccount_05_DeleteAccountConfirm
+    res.submitForm({
+      formSelector: "form[action='/delete-account']"
+    }),
+  { isStatusCode200, ...pageContentCheck('You have signed out') }))
   iterationsCompleted.add(1)
 }
 
 export function validateUser (): void {
+  const groups = groupMap.validateUser
   let res: Response
   const userData = validateData[exec.scenario.iterationInInstance % validateData.length]
   iterationsStarted.add(1)
 
-  res = group('B05_ValidateUser_01_LaunchAccountsHome GET', () =>
-    timeRequest(() => http.get(env.envURL, {
-      tags: { name: 'B05_ValidateUser_01_LaunchAccountsHome' }
-    }),
-    {
-      'is status 200': (r) => r.status === 200
-    }))
+  res = group(groups[0], () => timeRequest(() => // B05_ValidateUser_01_LaunchAccountsHome
+    http.get(env.envURL),
+  { isStatusCode200, ...pageContentCheck('Create a GOV.UK One Login or sign in') }))
 
   sleepBetween(1, 3)
 
-  res = group('B05_ValidateUser_02_ClickSignIn POST', () =>
-    timeRequest(() => res.submitForm({
-      params: { tags: { name: 'B05_ValidateUser_02_ClickSignIn' } }
-    }), { isStatusCode200, 'verify page content': r => (r.body as string).includes('Enter your email address to sign in to your GOV.UK One Login') }))
+  res = group(groups[1], () => timeRequest(() => // B05_ValidateUser_02_ClickSignIn
+    res.submitForm({
+      fields: {
+        supportInternationalNumbers: 'true'
+      }
+    }),
+  { isStatusCode200, 'verify page content': r => (r.body as string).includes('Enter your email address to sign in to your GOV.UK One Login') }))
```

Comment:
```suggestion
  { isStatusCode200, ...pageContentCheck('Enter your email address to sign in to your GOV.UK One Login') }))
```
Not necessarily in the scope of this PR, but just noticed this scenario is not using the `pageContentCheck()` utility function for most of the assertions.

This change can go in another PR, but just noting it here whilst reviewing these lines of code

Reply:
Thanks. This will be updated in the next PR. Leaving it as is for this PR